src/components/DateTimeField/index.jsx

```diff
@@ -11,7 +11,7 @@ class DateTimeField extends React.Component {
     super(props);
     this.state = {
       date: getDateString(this.props.input.value) || '',
-      time: getTimeString(this.props.input.value) || '',
+      time: getTimeString(this.props.input.value) || '12:00',
```

Comment:
Quick fix while I was in there, sets default form value to 12:00PM. Previous commit that addressed this only set the field's display text to 12:00PM.

Reply:
Good catch. Nice job!